Makefile

```diff
@@ -276,7 +276,6 @@ help: ## (Default) Print listing of key targets with their descriptions
 
 .PHONY: lint
 lint: golangci-lint
-	@echo "Linting vs commit '$(call err_if_empty,EPOCH_TEST_COMMIT)'"
```

Comment:
As best I can tell, `PRE_COMMIT` does not use `EPOCH_TEST_COMMIT`. This `echo` was added in #5066, back in February 2020, but I think it was a mistake.

Reply:
I'm pretty sure pre-cmmit doesn't.  I know vbatts/gitvalidate does use it though (in case that matters here).